backend/src/middleware/slugify/uniqueSlug.js

```diff
@@ -1,15 +1,19 @@
 import slugify from 'slug'
+
+let locale; 
+
+if(locale === 'de'){
+  locale = 'de'
```

Comment:
@Tirokk I have found that the slug package already contains a locale for German. It will convert the umlauts properly as well as ß -> ss. It will still use its standard charmap to remove the other diacritics. I was not able to figure out how to access the current locale to pass that into the slugify function to set the locale if Germany is selected.

Reply:
@Tirokk @Mogge The best universal option I have found for this would be the .normalize() for javascript. I need to read a bit more into the exact implementation issues but here is a sample I found. 

var str = 'áàâäãéèëêíìïîóòöôõúùüûñçăşţ';
var asciiStr = str.normalize('NFKD').replace(/[^\w]/g, '');
console.info(str, asciiStr);

We could then add an additional .replace  before the normalize to change the needed German characters. 

str.replace(#unicode replacements for umlauts and ß ).normalize('NFKD').replace(/[^\w]/g, '');